toolkit/scalebar/src/main/java/com/arcgismaps/toolkit/scalebar/Scalebar.kt

```diff
@@ -67,6 +68,49 @@ import kotlin.time.Duration
  * It provides a visual indication through which users can determine the size of features or
  * the distance between features on a map.
  *
+ * The required parameters to display a Scalebar are the maximum width of the Scalebar in device independent pixels (dp),
+ * the current unitsPerDip of the MapView, and the current [ViewpointType.CenterAndScale] of the MapView and the
+ * [SpatialReference] of the MapView. The current value of unitsPerDip can be obtained from the Composable
+ * MapView's onUnitsPerDipChanged callback. The [Viewpoint] can be obtained from the Composable MapView's
+ * onViewpointChangedForCenterAndScale callback and the [SpatialReference] can be obtained from the Composable MapView's
+ * onSpatialReferenceChanged callback.
+ *
+ * The Scalebar will be automatically updated when the Viewpoint changes.
+ *
+ * _Workflow example:_
+ *
+ *  ```
+ *     var viewpoint: Viewpoint? by remember { mutableStateOf(null) }
+ *     var unitsPerDip by remember { mutableDoubleStateOf(Double.NaN) }
+ *     var spatialReference: SpatialReference? by remember { mutableStateOf(null) }
+ *     // show composable MapView with a Scalebar
+ *     Box(
+ *         modifier = modifier.fillMaxSize()
+ *     ) {
+ *         MapView(
+ *             modifier = Modifier.fillMaxSize(),
+ *             arcGISMap = arcGISMap,
+ *             onSpatialReferenceChanged = { spatialReference = it },
+ *             onUnitsPerDipChanged = { unitsPerDip = it },
+ *             onViewpointChangedForCenterAndScale = { viewpoint = it }
+ *         )
+ *         Row(
+ *             modifier = Modifier
+ *                 .height(IntrinsicSize.Max)
+ *                 .fillMaxWidth()
+ *                 .padding(25.dp)
+ *                 .align(Alignment.BottomStart)
+ *         ) {
+ *             Scalebar(
+ *                 maxWidth = 175.dp,
+ *                 unitsPerDip = unitsPerDip,
+ *                 viewpoint = viewpoint,
+ *                 spatialReference = spatialReference,
+ *             )
+ *         }
```

Comment:
the code sample we use in the README is a bit simpler:
```suggestion
 *        Scalebar(
 *            modifier = Modifier
 *                .padding(25.dp)
 *                .align(Alignment.BottomStart),
 *            maxWidth = 175.dp,
 *            unitsPerDip = unitsPerDip,
 *            viewpoint = viewpoint,
 *            spatialReference = spatialReference,
 *        )
```
I propose we use the same code in `MainScreen.kt` for consistency.